tests/suite_utils.py

```
@@ -141,24 +145,56 @@
+            futures, return_when=concurrent.futures.FIRST_EXCEPTION
+        )
+        # Cancel remaining workers
+        for f in futures:
+            f.cancel()
+        # Check if any worker raised an unexpected error (not OOM)
+        for f in done:
+            exc = f.exception()
+            if exc is not None:
+                raise AssertionError(
+                    "Stress worker raised an unexpected error, OOM did not occur"
+                ) from exc
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)